api/caaCaseTypes/caaCaseTypes.util.ts

```diff
@@ -1,9 +1,21 @@
 import { CaaCasesPageType } from '../caaCases/enums';
 import { searchCasesString } from './caaCaseTypes.constants';
 
-export function getRequestBody(organisationID: string, caaCasesPageType: string) {
+export function getRequestBody(organisationID: string, caaCasesPageType: string, caaCasesFilterValue?: string | string[]) {
   const organisationAssignedUsersKey = `supplementary_data.orgs_assigned_users.${organisationID}`;
+  const reference = 'reference.keyword';
+  let caseReferenceFilter: any[] = [];
```

Comment:
@johnbenjamin-hmcts - `caseReferenceFilter` can be a `const`, if it's not being reassigned

Reply:
@Daniel-Lam - I have now updated the code.